lib/nerves/release.ex

```diff
@@ -6,6 +6,14 @@ defmodule Nerves.Release do
   def init(%{options: options} = release) do
     opts = Keyword.merge(options, release_opts())
 
+    vm_args = Mix.Release.rel_templates_path(release, "vm.args.eex")
+
+    if !File.exists?(vm_args) do
+      Mix.raise("""
+        rel/vm.args needs to be moved to #{vm_args}
+      """)
+    end
+
```

Comment:
I think this check can go away, this was a carry over from migrating from a distillery project to one that uses elixir releases. The rest of the distillery support was removed.